scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+matplotlib
+numpy
```

Comment:
You could add h5py here.

Reply:
https://github.com/unibas-dmi-hpc/SPH-EXA/pull/427/commits/6863a505e0d72c565097b9a9298c99fd5a9e701f